CHANGELOG.md

```diff
@@ -120,3 +124,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.1.0] - 2020-06-30
 ### Added
 - First release!
+
+[Unreleased]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v1.0.3...HEAD
+[1.0.3]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v1.0.2...v1.0.3
+[1.0.2]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v1.0.1...v1.0.2
+[1.0.1]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v1.0.0...v1.0.1
+[1.0.0]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v0.2.1...v1.0.0
+[0.2.1]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v0.2.0...v0.2.1
+[0.2.0]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v0.1.1...v0.2.0
+[0.1.1]: https://github.com/exop-group/doorkeeper-device_authorization_grant/compare/v0.1.0...v0.1.1 
+[0.1.0]: https://github.com/exop-group/doorkeeper-device_authorization_grant/releases/tag/v0.1.0
```

Comment:
Where's all that suddenly coming from?

Reply:
Oh I realized that we used "[links]" for version's heading, but they were not pointing to anything. That's a typical destination.

About this change, I admit I've been a little insolent and didn't bother to comment in the commit message... :innocent: